<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="4" hostname="Nitins-MacBook-Pro.local" failures="1" tests="1" name="Test_random(failed)" time="12.64" errors="0" timestamp="2024-02-20T23:14:18 PST">
  <testcase classname="com.salesforce.automation.SalesforceRandomScenarios_Testcases" name="blockingAnEventCalender" time="10.074">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.salesforce.automation.SalesforceRandomScenarios_Testcases.blockingAnEventCalender(SalesforceRandomScenarios_Testcases.java:317)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- blockingAnEventCalender -->
  <testcase classname="com.salesforce.automation.SalesforceRandomScenarios_Testcases" name="verifylastnameEdit_Variousplace" time="0.0">
    <ignored/>
  </testcase> <!-- verifylastnameEdit_Variousplace -->
  <testcase classname="com.salesforce.automation.SalesforceRandomScenarios_Testcases" name="weeklyrecurranceCalender" time="0.0">
    <ignored/>
  </testcase> <!-- weeklyrecurranceCalender -->
  <testcase classname="com.salesforce.automation.SalesforceRandomScenarios_Testcases" name="verifyAccountHolder" time="0.0">
    <ignored/>
  </testcase> <!-- verifyAccountHolder -->
  <testcase classname="com.salesforce.automation.SalesforceRandomScenarios_Testcases" name="verifytabcustomization" time="0.0">
    <ignored/>
  </testcase> <!-- verifytabcustomization -->
</testsuite> <!-- Test_random(failed) -->
